web/components/home/index.sass

```diff
@@ -1,6 +1,18 @@
 .homepage
   background: $gradient-purple
 
+  .mlh-trust-badge
+    display: block
+    max-width: 100px
+    min-width: 60px
+    position: absolute
+    right: 100px
+    top: 64px
+    width: 10%
+    z-index: 10000
+    @media screen and (max-width: $phablet)
```

Comment:
We have a utility class for this media query called [`scale-hide-phablet`](https://github.com/nwplus/nwhacks2019/blob/45867811c1a7e3c91118331723a3305901f07356/web/styles/breakpoints.sass#L71).